### PR TITLE
Feat/limit cart rules

### PR DESCRIPTION
**Pull request by Clap404**
# Separate endpoint to request cart rules.

Send cartrules in a pagniated form, and send them incrementally like other contents from the shop.
This avoids memory saturation on shops with a lot of rules and saves bandwidth by sending only modified rules. 

:warning: this PR solves the problem by adding a new endpoint, which needs to be added server-side too before publishing

### CODE REVIEW

File: e2e-env/.env.dist [hunk 1]
```diff
@@ -14,3 +14,6 @@ SYNC_API_PORT=3232
 COLLECTOR_API_PORT=3333
 LIVE_SYNC_API_PORT=3434
 WS_PORT=8080
+
+UID=1000
```
Comment (jokesterfr) [inline e2e-env/.env.dist:18]: Utile ?

File: e2e-env/cloudsync-mock/src/lib/server.ts [hunk 2]
```diff
@@ -9,20 +9,23 @@ export class Server {
 
   public constructor(port: number) {
```
Comment (jokesterfr) [inline e2e-env/cloudsync-mock/src/lib/server.ts:10]: Dans une PR à part ?

File: e2e-env/.env.dist (hunk 1 above)
Comment (Clap404) [reply e2e-env/.env.dist:18]: ok

**Comment by Clap404** [comment]
replaced by https://github.com/PrestaShopCorp/ps_eventbus/pull/229

**Pull request closed without merging**